php/libraries/Project.class.inc

```diff
@@ -75,9 +75,9 @@ class Project
 
                 $project = new Project();
 
-                $project->projectId         = $projectData['projectId'];
-                $project->projectName       = $projectData['name'];
-                $project->recruitmentTarget = (integer) $projectData['recTarget'];
+                $project->_projectId         = $projectData['projectId'];
```

Comment:
The class variable is defined as _projectId, but the code was referring to projectId. This was a bug.

I had to change the code and not the unused variable name, or the PHPCS rules would complain.